### PR TITLE
feat(config): make check for frontend instead of gms more robust

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-ingestion/src/datahub/emitter/rest_emitter.py
Comment (shirshanka) [inline metadata-ingestion/src/datahub/emitter/rest_emitter.py:77]: Not necessarily - going forward, datahub-frontend can serve as a proxy to GMS. People can point to datahub-frontend instead to minimize the access points they need to maintain.

File: metadata-ingestion/src/datahub/emitter/rest_emitter.py
Comment (shirshanka) [inline metadata-ingestion/src/datahub/emitter/rest_emitter.py:137]: This does not need to be so strict

File: metadata-ingestion/src/datahub/emitter/rest_emitter.py
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/emitter/rest_emitter.py:129]: Not necessarily - going forward, datahub-frontend can serve as a proxy to GMS. People can point to datahub-frontend instead to minimize the access points they need to maintain.

**Review by shirshanka — APPROVED** [review]
LGTM. Will wait for CI before merging.

**Pull request merged**
